src/operator/nn/mkldnn/mkldnn_transpose.cc

```diff
@@ -121,6 +107,14 @@ class MKLDNNTransposeForward {
   const mkldnn::reorder &GetFwd() const {
     return *transpose_;
   }
+
+  void Execute() const {
+    auto stream = MKLDNNStream::Get();
+    std::unordered_map<int, mkldnn::memory> net_args;
```

Comment:
Please use `mkldnn_args_map_t` instead of `std::unordered_map<int, mkldnn::memory>`.

Reply:
Fixed thanks